src/porepy/utils/txt_io.py

```diff
@@ -0,0 +1,96 @@
+"""Module for exporting/importing numpy arrays to/from txt files."""
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import Type
+
+import numpy as np
+
+
+@dataclass
+class TxtData:
+    """Class for the representation of txt data."""
+
+    header: str
+    """Name of the data."""
+
+    array: np.ndarray
+    """Value of the data."""
+
+    format: str = "%2.2e"
+    """Format of the array values to be included in the txt file."""
+
+
+def export_data_to_txt(
+    list_of_txt_data: list[TxtData],
+    file_name: str = "out.txt",
+) -> None:
+    """Write data into a txt file.
+
+    Raises:
+        ValueError
+            If the sizes of the arrays from the `list_of_txt_data` are not equal.
+
+    Parameters:
+        list_of_txt_data: List of txt data.
+        file_name: ``default="out.txt"`
+
+            Name of the txt file to be exported.
+
+    """
+    # Sanity check
```

Comment:
My impression is that we have agreed to use complete sentences (here, use a full stop). This is far from crucial, but let's make sure that we agree on the guidelines.

Reply:
Ok.